frontend/apps/marketing/cicd/3-app/template.yml.erb

```
@@ -755,3 +755,9 @@ Outputs:
   SiteDomainName:
     Description: Fully qualified domain name of the marketing site
     Value: !Sub "${SubdomainName}.${BaseDomainName}"
+  CloudFrontDistributionId:
+    Value:
+      Ref: CloudFrontDistribution
+    Export:
+      Name:
+        'Fn::Sub': '${AWS::StackName}-CloudFrontDistributionId'
```

GitHub Actions is getting this value by fetching the Ouputs of the Stack and we're not Importing this value in any other Stack, so we could just return an Output, not Export it, and to be consistent with the rest of the template, use the slightly more readable `!Ref` syntax.
```suggestion
    Value: !Ref CloudFrontDistribution
```